fastai/tabular/data.py

```diff
@@ -12,22 +12,39 @@ class TabularDataLoaders(DataLoaders):
     "Basic wrapper around several `DataLoader`s with factory methods for tabular data"
     @classmethod
     @delegates(Tabular.dataloaders, but=["dl_type", "dl_kwargs"])
-    def from_df(cls, df, path='.', procs=None, cat_names=None, cont_names=None, y_names=None, y_block=None,
-                valid_idx=None, **kwargs):
-        "Create from `df` in `path` using `procs`"
+    def from_df(cls,
+                df:pd.DataFrame, # A Dataframe object
+                path:str='.', # Location of `df` (defaults to current working directory otherwise)
+                procs:TabularProc=None, # List of `TabularProc`s (non-lazy `Transform`s) applied on object inplace
+                cat_names:list=None, # Column names pertaining to Categorical variables
+                cont_names:list=None, # Column names pertaining to Continuous variables
+                y_names:list=None, # Names of the Dependent variables
+                y_block:TransformBlock=None, # `TransformBlock` to use for the Target
+                valid_idx:list=None, # List of indices to use for the Validation set (defaults to a random split otherwise)
+                **kwargs):
+        "Create `DataLoaders` from `df` in `path` using `procs`"
```

Comment:
```suggestion
        "Create `TabularDataLoaders` from `df` in `path` using a list of `TabularProc`s"
```
1. I think this actually creates a TabularDataLoaders
2. Thoughts on passing TabularProc here instead of procs so that it links to that section of docs?

Reply:
Perfect, Accepted 👍